src/Altinn.Platform/Altinn.Platform.Authorization/Authorization/Repositories/Interface/IPolicyInformationRepository.cs

```diff
@@ -0,0 +1,34 @@
+using System.Threading.Tasks;
+using Altinn.Platform.Storage.Models;
+
+namespace Altinn.Platform.Authorization.Repositories.Interface
+{
+    /// <summary>
+    /// Interface for operations on instance information
+    /// </summary>
+    public interface IPolicyInformationRepository
+    {
+        /// <summary>
+        /// Gets the information of a given instance
+        /// </summary>
+        /// <param name="instanceId">the instance id</param>
+        /// <param name="instanceOwnerId">the instance owner</param>
+        /// <returns></returns>
+        Task<Instance> GetInstance(string instanceId, int instanceOwnerId);
```

Comment:
should this be named instanceGuid or is it supposed to be on the form {instanceOwner}/{instanceguid} ?

Reply:
I just remember the tech archs were not very happy to use the datatype at the end of the variable